[2024-08-08T06:07:23.786+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-08T06:07:23.807+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_2024-08-08.log_the_data scheduled__2024-05-22T00:00:00+00:00 [queued]>
[2024-08-08T06:07:23.812+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_2024-08-08.log_the_data scheduled__2024-05-22T00:00:00+00:00 [queued]>
[2024-08-08T06:07:23.812+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-08T06:07:23.880+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): log_the_data> on 2024-05-22 00:00:00+00:00
[2024-08-08T06:07:23.892+0000] {standard_task_runner.py:63} INFO - Started process 3497 to run task
[2024-08-08T06:07:23.897+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_2024-08-08', 'log_the_data', 'scheduled__2024-05-22T00:00:00+00:00', '--job-id', '1156', '--raw', '--subdir', 'DAGS_FOLDER/final_etl.py', '--cfg-path', '/tmp/tmpzyhqvmc5']
[2024-08-08T06:07:23.899+0000] {standard_task_runner.py:91} INFO - Job 1156: Subtask log_the_data
[2024-08-08T06:07:23.986+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_2024-08-08.log_the_data scheduled__2024-05-22T00:00:00+00:00 [running]> on host 05bc191a9412
[2024-08-08T06:07:24.119+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_2024-08-08' AIRFLOW_CTX_TASK_ID='log_the_data' AIRFLOW_CTX_EXECUTION_DATE='2024-05-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-22T00:00:00+00:00'
[2024-08-08T06:07:24.121+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-08T06:07:24.136+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-08T06:07:24.137+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/data/logging_config.py']
[2024-08-08T06:07:24.145+0000] {subprocess.py:86} INFO - Output:
[2024-08-08T06:07:24.384+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-08T06:07:24.385+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-08T06:07:24.432+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_2024-08-08, task_id=log_the_data, run_id=scheduled__2024-05-22T00:00:00+00:00, execution_date=20240522T000000, start_date=20240808T060723, end_date=20240808T060724
[2024-08-08T06:07:24.468+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-08T06:07:24.490+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-08T06:07:24.515+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
